### PR TITLE
Linear Model Coefficient Analysis

### PR DESCRIPTION
**Pull request by jenna-tomkinson**
# Linear Model Coefficient Analysis

In this PR, I include:

1. Updated README to include a link to ML repository
2. Perform LM analysis per CellProfiler feature and generate scatter plot per plate

# What I would like in a review

There are a total of **286 lines of code** to review across READMEs, python and R scripts, env, and gitignore files.

Please feel free to provide suggestions regarding documentation and improving the code.

### CODE REVIEW

File: 4.analyze_data/notebooks/linear_model/scripts/1.visualize_lm.r [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+            color = guide_legend(title = "Channel\n(if applicable)", order = 1),
+            size = guide_legend(title = "R2 score")
+        )
+        +
+        ylab("WT genotype contribution (LM beta coefficient)")
+        +
+        xlab("Cell count contribution (LM beta coefficient)")
+        +
+        ggtitle("Scatter plot of linear model coefficients per CellProfiler feature\nshows ER features are most differential between WT and Null") # nolint
+)
```
Comment (axiomcura) [inline 4.analyze_data/notebooks/linear_model/scripts/1.visualize_lm.r]: I would suggest providing a simple comment explaining what is being plotted.

**Review by axiomcura — APPROVED** [review]
LGTM, just left a small comment about the plot function in R

File: 4.analyze_data/notebooks/linear_model/scripts/1.visualize_lm.r (hunk 1 above)
Comment (jenna-tomkinson) [reply 4.analyze_data/notebooks/linear_model/scripts/1.visualize_lm.r]: Sounds good! I have provided a bit more info in code comments. Thanks for pointing this out!

**Comment by jenna-tomkinson** [comment]
@axiomcura Thanks Erik for the review! I will be merging now!

**Pull request merged**